tests/InputData.json

```
@@ -0,0 +1,58 @@
+{		
+	"SDDCManager" : "sfo-vcf01.sfo.rainpole.io",
+	"SDDCManagerVMName" : "sfo-vcf01",
+	"User"	:	"administrator@vsphere.local",
+	"RootUser"	: "root",
+	"GuestUser" : "root",
+	"LocalUser" : "vcf",
+	"Password" 	: "VMw@re1!",
+	"SsoAdmin"	: "admin@local",
+	"SsoPass"	: "VMw@re1!VMw@re1!",
+	"RootPass"	: "VMw@re1!",
+	"Domain"	: 	[
+						"sfo-w01",
+						"sfo-m01"
+					],
+	"sfo-w01": {
+		"Esxi Server" : [
+							"sfo01-w01-esx01.sfo.rainpole.io",
+							"sfo01-w01-esx02.sfo.rainpole.io",
+							"sfo01-w01-esx03.sfo.rainpole.io",
+							"sfo01-w01-esx04.sfo.rainpole.io"
+						],
+		"Edge Nodes" : [
+							"sfo-w01-en01",
+							"sfo-w01-en02"
+		],
+		"Manager Node" : [
+			"sfo-w01-nsx01.sfo.rainpole.io"
+		],
+		"Manager Nodes" : [
+				"sfo-w01-nsx01a.sfo.rainpole.io",
+				"sfo-w01-nsx01b.sfo.rainpole.io",
+				"sfo-w01-nsx01c.sfo.rainpole.io"
+		],
+		"Cluster"	:	["sfo-w01-cl01"]
+	},
+	"sfo-m01": {
+		"Esxi Server" : [
+			"sfo01-m01-esx01.sfo.rainpole.io",
+			"sfo01-m01-esx02.sfo.rainpole.io",
+			"sfo01-m01-esx03.sfo.rainpole.io",
+			"sfo01-m01-esx04.sfo.rainpole.io"
+		],
+		"Edge Nodes" : [
```

```suggestion
		"NSX Edge Nodes" : [
```

Could we update this?